src/studio/README.md

```diff
@@ -78,6 +78,8 @@ Example
 docker-compose up -d --build altinn_designer
 ```
 
+Next you just have to create a new user and start testing. (No email verification required)
```

Comment:
How about moving this closer to line 67 where the url for the local solution is presented?

Reply:
Seems reasonable